File: oop_6/view/StudentView.java
package oop_6.view;

import oop_6.data.Student;
import oop_6.data.StudentGroup;

import java.util.List;
import java.util.logging.Logger;

public class StudentView implements UserView<Student> {

    Logger logger = Logger.getLogger(StudentView.class.getName());

    @Override
    public void sendOnConsole(List<Student> students) {
        for (Student user : students) {
            logger.info(user.toString());
        }
    }

    public void sendOnConsoleUserGroup(StudentGroup studentGroup) {
        logger.info(studentGroup.toString());
    }

}

File: intermediate_certification_work/IntermediateCertWorkConsoleApp/IntermediateCertWorkConsoleApp/view/StudentView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

public class StudentView : IUserView<Student>
{
    public void sendOnConsole(List<Student> students)
    {
        foreach (Student user in students)
        {
            Console.WriteLine(user.ToString());
        }
    }
}